types.js

```diff
@@ -195,19 +195,49 @@ type _SJLayerBase = {
   resizingType?: ResizingType,
 } & SJIDBase;
 
-export type SJLayer = SJArtboardLayer | SJTextLayer | SJGroupLayer | SJShapeGroupLayer | SJShapeLayer;
-
-export type SJArtboardLayer = {
-  _class: 'artboard',
+type _SJArtboardBase = {
   frame: SJRect,
   backgroundColor: SJColor,
   hasBackgroundColor: bool,
   horizontalRulerData?: RulerData,
   verticalRulerData?: RulerData,
   includeBackgroundColorInExport?: bool,
-  includeInCloudUpload?: bool,
+  includeInCloudUpload?: bool
+} & _SJLayerBase;
+
+export type SJLayer = SJArtboardLayer | SJTextLayer | SJGroupLayer | SJShapeGroupLayer | SJShapeLayer | SJSymbolInstanceLayer;
+
+export type SJSymbolMaster = {
+  _class: 'symbolMaster',
+  includeBackgroundColorInInstance?: bool,
+  symbolID: string
+} & _SJArtboardBase;
+
+export type SJNestedSymbolOverride = {
+  symbolID: string
+}
+
+export type SJSymbolInstanceLayer = {
+  _class: 'symbolInstance',
+  frame: SJRect,
+  horizontalSpacing: number,
+  verticalSpacing: number,
+  masterInfluenceEdgeMinXPadding?: number,
+  masterInfluenceEdgeMaxXPadding?: number,
+  masterInfluenceEdgeMinYPadding?: number,
+  masterInfluenceEdgeMaxYPadding?: number,
+  symbolID: string,
+  overrides?: {
+    '0': {
+      [objectId: string]: string | SJNestedSymbolOverride | SJImageDataReference
```

Comment:
It would probably be a good idea to make a type for object ids besides `string` but don't worry about it for this PR